config/lookout/config.yaml

```diff
@@ -4,6 +4,7 @@ metricsPort: 9009
 
 uiConfig:
   armadaApiBaseUrl: "http://localhost:8080"
+  annotationPrefix: "armada/"
```

Comment:
lets go for `armadaproject.io/` to follow the domain pattern other things use with annotations.